backends-clickhouse/src/test/scala/org/apache/gluten/execution/tpch/GlutenClickHouseTPCHSaltNullParquetSuite.scala

```diff
@@ -3463,5 +3463,31 @@ class GlutenClickHouseTPCHSaltNullParquetSuite extends GlutenClickHouseTPCHAbstr
     compareResultsAgainstVanillaSpark(sql5, true, { _ => })
 
   }
+
+  test("GLUTEN-8974 accelerate join + aggregate by any join") {
```

Comment:
It's better to check the plan whether the agg + join is transformed to any join ?